src/server/raft_client.rs

```diff
@@ -29,7 +29,7 @@ use super::metrics::*;
 
 const MAX_GRPC_RECV_MSG_LEN: usize = 10 * 1024 * 1024;
 const MAX_GRPC_SEND_MSG_LEN: usize = 10 * 1024 * 1024;
-const INITIAL_BUFFER_CAP: usize = 1024;
+const MAX_MSG_BUFFER_COUNT: usize = 1024;
```

Comment:
I don't think it's the `MAX` count. The count of messages in buffer can exceed 1024.

Reply:
If the size of vector exceed 1024, the new pushing message operation to vector will cause reallocation, it's not like what we mean preserved buffer. CAP makes me confused, can it exceed CAP?

Reply:
It can exceed CAP, like Go `make([]byte, 0, 10)`, we can use an initialized capacity, but the slice can still grow exceeding it.